sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -393,7 +393,7 @@ object LoadDataCommand {
           throw new IllegalArgumentException(e)
       }
     } else {
-      path
+      new Path(pathUri)
```

Comment:
`path` doesn't contain `workingDir` information.

Could you review this, @cloud-fan , @gatorsmile , @HyukjinKwon , @vanzin and @squito ?

Reply:
Here, it is converted PATH to URI and then converted back to Path. What is your goal rather than directly building a path?
```
if (path.isAbsolute()) path else new Path(workingDir, path)
````

Reply:
Good point. Instead of this, `new Path(workingDir, path)` will work. In that case, we may refactor this as a variable for line 379 and reuse it.
```
new Path(workingDir, path)
```